openmetrics/src/main/java/io/airlift/openmetrics/MetricsResource.java

```diff
@@ -202,7 +204,7 @@ private Optional<Metric> getMetric(ObjectName objectName, String attributeName,
             }
 
             if (attributeValue instanceof Number) {
-                return Optional.of(Gauge.from(metricName, (Number) attributeValue, description));
+                return Optional.of(Gauge.from(metricName, List.of(), (Number) attributeValue, description));
```

Comment:
This is clearly wrong, but I'm not sure what's the purpose of constructing metrics from advanced filters with these separators like `_ATTRIBUTE_`.